e2e-tests/playwright/package.json

```
@@ -22,6 +22,7 @@
         "async-wait-until": "2.0.12",
         "axe-core": "4.10.0",
         "chalk": "4.1.2",
+        "dayjs": "^1.11.13",
```

nit: let's use fix version.
```suggestion
        "dayjs": "1.11.13",
```